data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.linkManager/1.0/corda.p2p.linkManager.json

```diff
@@ -171,12 +171,6 @@
       "type": "object",
       "default": {},
       "properties": {
```

Comment:
You can't remove config properties, see https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4564091182/Making+a+Config+Change#Not-Permitted-Changes

The reason being 5.3 Corda will read 5.2 config on startup after an upgrade, it will see this property from the existing config and flag it's an invalid property not in its schema and fail.